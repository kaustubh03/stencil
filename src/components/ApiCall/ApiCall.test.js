import React from "react";
import renderer from "react-test-renderer";
import Apicall from ".";
import configureStore from "../../redux/configureStore";
const store = configureStore();

it("Apicall: default", () => {
  const component = renderer.create(<Apicall store={store} />);
  const tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});
