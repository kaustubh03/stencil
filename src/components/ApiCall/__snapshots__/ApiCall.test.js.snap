// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Apicall: default 1`] = `
<div>
  <div
    className="panelClass panel panel-default"
  >
    <h2
      className={null}
    >
      API Calls with Redux, Axios
    </h2>
    <div
      className="btn-toolbar"
      role="toolbar"
    >
      <button
        className="btn btn-success"
        disabled={false}
        onClick={[Function]}
        type="button"
      >
        Fetch Data From Sample API
      </button>
    </div>
    <div
      className="panel-body"
    >
      <ul
        className="list-group"
      >
        Sorry!!No Data Here.
      </ul>
    </div>
  </div>
</div>
`;
